[DOCS] Update and clarify descriptions, context, and syntax in Actions-List.md (#3752)
BetonQuest/BetonQuest: docs/Documentation/Scripting/Building-Blocks/Actions-List.md

  sendPluginCommand: "chat /someCommand x y z"
```

## Clear a chest

__Context__: @snippet:event-meta:independent@  
__Syntax__: `chestclear <location>`  

> __Description__: This action removes all items from a chest at specified location. The only argument is a location.

Reviewer: ```suggestion
__Description__: Remove all items from a chest at the specified location.
```

```YAML title="Example"
actions:
  chestclear: "chestclear 100;200;300;world"
```

## Put items in a chest

__Context__: @snippet:event-meta:independent@  
__Syntax__: `chestgive <location> <items>`  

> __Description__: Puts items in a chest at specified location.

Reviewer: ```suggestion
__Description__: Put items in a chest at the specified location.
```

  chesttake: "chesttake 100;200;300;world emerald:5,sword"
```

> ## Control the compass targets

Reviewer: ```suggestion
## Modify compass targets
```

__Context__: @snippet:event-meta:online-offline@  

> __Syntax__: `chesttake <operation> <target>`  

Reviewer: Thats no longer the compass event.

  killAndBan: "command kill %player%|ban %player%"
```

## Start a conversation

__Context__: @snippet:event-meta:online@  
__Syntax__: `conversation <conversation> <option>`  

> __Description__: Starts a conversation.

Reviewer: ```suggestion
__Description__: Start a conversation.
```

  startConversationOption: "conversation tutorial option:explain_world"
```

## Damage the player

__Context__: @snippet:event-meta:online@  
__Syntax__: `damage <amount>`  
__Description__: Damages the player by specified amount of damage.

The only argument is a number (can have a floating point).

```YAML title="Example"
actions:
  dealDamage: "damage 20"
```

## Delete points

__Context__: @snippet:event-meta:online-offline@  
__Syntax__: `deletepoint <category>`  

> __Description__: Clear all player points in a specified category.

Reviewer: ```suggestion
__Description__: Clear all points for a player in the specified category.
```

  kickAll: "runForAll where:!isOp actions:kickPlayer,restartQuest"
```

> ## Run actions as independent

Reviewer 2: That sounds awful and does not make sense gramatically.
I suggest to just remove the as or even ad the `ly` at the end.
```suggestion
## Run actions independently
```
